nendo_server/apps/polymath/routes.py

```diff
@@ -49,6 +49,9 @@ async def run_polymath(
             container_name="",
             exec_run=False,
             replace_plugin_data=replace,
+            run_without_target=False,
+            max_track_duration=900.,  # 15 minutes per track
```

Comment:
I think all these values should be in environment variables probably, so that we can quickly change them on deployments if necessary? What do you think?

Reply:
Yes, I had the same thought. However, I think this would require us to make a move in the direction of complete "tools manifests", since these values vary greatly between tools and introducing three new config vars for each tool sounds a bit messy to me. Let's discuss

Reply:
yeah that sounds like too much, you're right. then let's keep the defaults for now, and maybe add a backlog task for adding "manifests" - I can imagine having a `pydantic.Config` for each tool maybe? similar to what the plugins currently do, then we get all the ENV stuff for free if we need it.